packages/react/src/utils/felaRenderer.tsx

```diff
@@ -67,12 +67,12 @@ const rendererConfig = {
     felaPluginEmbedded(),
     felaPluginPrefixer(),
 
+    felaExpandCssShorthandsPlugin(),
```

Comment:
This move is valid as `inline-style-expand-shorthand` handles arrays on its side.

Reply:
How are we handling arrays exactly? As far as I can see, we are just spreading the array:
```
        if (Array.isArray(cssPropertyValue)) {
          return { ...acc, [cssPropertyName]: cssPropertyValue }
        }
```

Is the `expandProperty` function handling arrays?

Reply:
`expandProperty` exactly, I added this functionality in https://github.com/robinweser/inline-style-expand-shorthand/pull/13